proposals/dao/tip_121c.ts

```diff
@@ -23,7 +24,6 @@ const AMOUNT_FEI_MINTED_BY_E2E = toBN('10000000000000000000000000'); // 10M
 
 /////////////  Tribe Redeemer config
 // Circulating amount of TRIBE, which is redeemable for underlying PCV assets
-// TODO: Update with final circulating TRIBE figure
 const REDEEM_BASE = ethers.constants.WeiPerEther.mul(458_964_340);
```

Comment:
This number hasn't changed; was it supposed to?

Reply:
No, it's fixed and is an invariant

Reply:
It's calculated by tribe.totalSupply() - a bunch of tribe locked in various places like Core